bindings/Makefile

```diff
@@ -0,0 +1,37 @@
+SWAGGER_SPEC := ../proto/build/swagger/determined/api/v1/api.swagger.json
+CODEGEN_BIN := swagger-codegen-cli-2.4.14.jar
+
+.PHONY: all
+all: get-deps
+	$(MAKE) build
+
+.PHONY: get-deps
+get-deps: deps/${CODEGEN_BIN}
+
+deps/${CODEGEN_BIN}:
+	mkdir -p deps
+	curl https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.14/${CODEGEN_BIN} \
+		-o deps/${CODEGEN_BIN}
+
+build/%: ${SWAGGER_SPEC} deps/${CODEGEN_BIN}
+	mkdir -p build/$*
+	java -jar deps/$(CODEGEN_BIN) generate -i ${SWAGGER_SPEC} \
+		-l $* -o build/$*
+	touch build/$*
+
+.PHONY: build-default
+build-default: build/typescript-fetch build/python
```

Comment:
Whats the purpose of this target?

Reply:
it was to allow different bindings to be built concurrently with -j but I needed to do this https://github.com/determined-ai/determined/pull/2011/commits/9d999f52620ec6bb3e8a7f1c5e15a76f1bd391d3